WeakAuras/Prototypes.lua

```diff
@@ -1621,7 +1621,7 @@ WeakAuras.event_prototypes = {
         type = "unit",
         init = "arg",
         values = "actual_unit_types_with_specific",
-        test = "event ~= 'UNIT_POWER_FREQUENT' or UnitIsUnit(unit, '%s' or '')"
+        test = "event ~= 'UNIT_POWER_FREQUENT' or UnitIsUnit(unit, [[%s]] or '')"
```

Comment:
Also string values are truthy even when empty so the or at the end is redundant (I know that you didn’t introduce this change pls don’t shoot me)